environment/k8s_environment_helm_test.go

```diff
@@ -39,7 +39,7 @@ var _ = Describe("Environment with Helm @helm_deploy", func() {
 			Expect(err).ShouldNot(HaveOccurred())
 		})
 		AfterEach(func() {
-			By("Tearing down the environment", func() {
```

Comment:
There's a lot of environment > env changes which will have been done during a Goland refactor, can these comment/string changes be reverted